.travis.yml

```diff
@@ -5,59 +5,26 @@ env:
        dist: trusty
        language: generic
+services:
+  - docker
 
-before_install:
-   - until sudo add-apt-repository -y ppa:saiarcot895/chromium-beta; do echo retry; done
-   - until sudo apt-get -qq update; do echo retry; done
-   - until sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test; do echo retry; done
-   - sudo apt-add-repository --yes ppa:smspillaz/cmake-2.8.12
-   - sudo apt-get --yes update
-   - sudo apt-get install --yes cmake cmake-data
-   - sudo apt-get install --yes python-dev
-   - sudo apt-get install --yes python-pip
-   - sudo pip install lit
-   - sudo pip install OutputCheck
-   - sudo easy_install networkx
-   - sudo apt-get install libgraphviz-dev
-   - sudo easy_install pygraphviz
-
-install:
-  - until sudo apt-get -qq update; do echo retry; done
-  - until sudo apt-get -qq install libstdc++5; do echo retry; done
-  - until sudo apt-get -qq install g++-5; do echo retry; done
-  - until sudo apt-get install --yes libgmp-dev; do echo retry; done
-  - until sudo apt-get install --yes libmpfr-dev; do echo retry; done
-  - until sudo apt-get install -qq clang-3.8; do echo retry; done
-  ### precompiled binaries for LLVM 3.8
-  - export LLVM_ROOT="$TRAVIS_BUILD_DIR/LLVM-3.8-Linux"
-  - wget https://jorgenavas.github.io/sw/LLVM-3.8-Linux.tar.gz  
-  - tar xf LLVM-3.8-Linux.tar.gz  
-  ### precompiled binaries for z3
-  - export Z3_ROOT="$TRAVIS_BUILD_DIR/Z3-Linux"
-  - wget https://jorgenavas.github.io/sw/Z3-Linux.tar.gz
-  - tar xf Z3-Linux.tar.gz
-  ### BOOST
-  - wget --no-verbose --output-document=boost-trunk.tar.bz2 https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.bz2
-  - export BOOST_ROOT="$TRAVIS_BUILD_DIR/boost-trunk"
-  - export CMAKE_MODULE_PATH="$BOOST_ROOT"
-  - mkdir -p $BOOST_ROOT
-  - tar jxf boost-trunk.tar.bz2 --strip-components=1 -C $BOOST_ROOT
-  - (cd $BOOST_ROOT; ./bootstrap.sh --with-libraries=date_time,filesystem,system,iostreams,regex,test)
-  - (cd $BOOST_ROOT; ./b2 threading=multi --prefix=$BOOST_ROOT -d0 install)
+install: true
 
 script:
-  - mkdir -p build
-  - cd build
-  - /usr/bin/cmake -DBOOST_ROOT=${BOOST_ROOT} -DZ3_ROOT=${Z3_ROOT} -DLLVM_DIR=${LLVM_ROOT}/share/llvm/cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=g++-5 -DCMAKE_PROGRAM_PATH=/usr/bin  -DCMAKE_INSTALL_PREFIX=run ../;
-  ## XXX: not required since LLVM and Z3 are already found
-  #- /usr/bin/cmake --build .
-  - /usr/bin/cmake --build . --target extra && /usr/bin/cmake ..
-  - /usr/bin/cmake --build . --target crab && /usr/bin/cmake ..
-  - /usr/bin/cmake --build . --target install
-  - /usr/bin/cmake --build . --target test-simple
-  - /usr/bin/cmake --build . --target test-solve
-  - /usr/bin/cmake --build . --target test-abc   
-  - /usr/bin/cmake --build . --target test-dsa
+  ### build with docker
+  - docker build --build-arg UBUNTU=trusty --build-arg BUILD_TYPE=MinSizeRel --build-arg TRAVIS=true -t seahorn_trusty_build -f docker/seahorn-full-size-rel.Dockerfile .
```

Comment:
why are there 2 `docker build` commands? I was expecting only one

Reply:
is the first command for the build and the second to create a seahorn distribution to run the test?
If so, please comment this in the file to avoid similar confusion in the future.